config/config.go

```diff
@@ -75,6 +77,8 @@ func ConfigurationFromEnvironment() *Configuration {
 		ret.LicenseKey = licenseKey
 	} else if lkSecretOverride {
 		ret.LicenseKeySecretId = licenseKeySecretId
+	} else if lkSSMParameterOverride {
```

Comment:
I tested this in an environment with a license key in secrets manager and an SSM key, and it seems like the SSM key gets overridden if there is a key in secrets manager with the name "NEW_RELIC_LICENSE_KEY". If a customer chooses to provide a key through an environment variable explicitly, then we should always choose to use that key. Is there a way you could add that behavior to this code?

Reply:
The intended behavior you described should be how this already works in `GetNewRelicLicenseKey`. I haven't tested this exact scenario myself, I'll try to reproduce what you saw.

Reply:
@iamemilio Based on my testing, as long as the `NEW_RELIC_LICENSE_KEY_SSM_PARAMETER_NAME` environment variable exists, that takes precedence over a Secrets Manager secret or an SSM parameter with the `NEW_RELIC_LICENSE_KEY` name. So I believe this code is already working as you requested.